config/BoardConfigCM.mk

```diff
@@ -1,7 +1,7 @@
 # Charger
-#ifneq ($(WITH_CM_CHARGER),false)
-#    BOARD_HAL_STATIC_LIBRARIES := libhealthd.cm
-#endif
+ifneq ($(WITH_CM_CHARGER),false)
```

Comment:
This essentially undoes acf0d238dc44602272b8d9e1b3c986ed75bd6e0a, which might not be desired, please reattempt the merge.